app/src/main/java/fraktal/io/android/demo/timer/ui/TimerView.kt

```diff
@@ -30,21 +30,25 @@ import androidx.compose.ui.unit.sp
 import fraktal.io.android.demo.timer.domain.TimerCommand
 import fraktal.io.android.demo.timer.domain.TimerEvent
 import fraktal.io.android.demo.timer.domain.TimerState
-import fraktal.io.ext.Reducer
+import fraktal.io.ext.EventBus
+import fraktal.io.ext.Aggregate
+import fraktal.io.ext.MaterializedView
 import kotlinx.coroutines.launch
 
 
 @Composable
 fun TimerView(
```

Comment:
Great. Perhaps this option is more appealing to me (than the other one). Should we focus on it instead?
I would like to make a small commit later and need to add some Android-specific code.

Reply:
Yes, this is better. Agree! We should focus on this, yes.

Reply:
@idugalic and more. View in the android world is an association with the UI level. can we assign another alias to denote this data type? FView or Presentation. Otherwise it will complicate the understanding

Reply:
Yes, sure.

Reply:
You can call it FView, Read, Query, or Presentation. Whatever makes sense to you. I am happy whit any, as long as we have this VIew in our domain model!